zerver/openapi/zulip.yaml

```diff
@@ -20448,12 +20448,15 @@ paths:
 
                         Administrators can always administer a channel.
 
-                        Note that a user must also [have access](/help/channel-permissions) to a
-                        channel in order to modify it. The exception to this rule is that
-                        organization administrators can edit channel names and descriptions
-                        without having full access to the channel.
+                        Note that a user must [have access](/help/channel-permissions) to a
+                        channel in order to add other subscribers to the channel.
```

Comment:
"add other subscribers to the channel, edit who can add other subscribers, make the channel public, or otherwise take actions that could grant access to the channel's message content."

I imagine we'll do a pass to use the "content access" terminology soon, but it seems worth spelling out the list.